tapdb/assets_store.go

```diff
@@ -1153,7 +1153,8 @@ func (a *AssetStore) FetchAssetProofs(ctx context.Context,
 		// TODO(roasbeef): can modify the query to use IN somewhere
 		// instead? then would take input params and insert into
 		// virtual rows to use
-		for _, locator := range targetAssets {
+		for ind := range targetAssets {
```

Comment:
Was this from an older linter before the new scoping rules became the default?

Reply:
It's from gosec rule G601, but the new rules won't apply for this project for a while since we're pinned to go 1.20. With the `GOEXPERIMENT=loopvar` build option we should already be fine but would rather leave this lint on anyways.